exersice_html/home.html

```diff
@@ -0,0 +1,239 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Shopka</title>
+</head>
+
+<body>
+  <header class="my-header">
+    <div class="header-wrap">
+      <div class="header-left">
```

Comment:
should be same `header` as initial word (not `my-header`), will be useful when using scss/sass